tests/helm/terraform/static/all_fields.output.yaml

```diff
@@ -39,18 +39,24 @@ data:
     #
     # shellcheck disable=SC2010
     # extract target directory names from the file names using _ as separator
+  
+    set -euo pipefail
+  
     err_report() {
         echo "Custom script error on line $1"
         exit 1
     }
     trap 'err_report $LINENO' ERR
   
-    for dir in $(ls -1 /customer-scripts | grep _ | grep -oE '^.*?_' | sed 's/_//g' | sort | uniq); do
+    declare -a dirs
+    ls -1 /customer-scripts 2>/dev/null | grep _ | grep -oE '^.*?_' | sed 's/_//g' | sort | uniq | read -ar dirs || true
+    for dir in "${dirs[@]}"; do
       target="/scripts/${dir}"
       mkdir "${target}"
       # shellcheck disable=SC2010
       # Get files for given directory and take only filename part (after first _)
-      for file in $(ls -1 "/customer-scripts/${dir}_"* | grep -oE '_.*' | sed 's/_//g'); do
+      files=$(ls -1 "/customer-scripts/${dir}_"* | grep -oE '_.*' | sed 's/_//g')
+      for file in ${files}; do
```

Comment:
Why do we behave differently for for files than for dirs?
```
declare -a dirs
...
```

Reply:
Shellcheck complains about one, but not the other, not sure why.

Reply:
I fixed one and forgot about the other. Done now.